src/main/java/com/google/devtools/build/lib/remote/RemoteExecutionService.java

```diff
@@ -585,8 +589,8 @@ public RemoteAction buildRemoteAction(Spawn spawn, SpawnExecutionContext context
       ActionKey actionKey = digestUtil.computeActionKey(action);
 
       RequestMetadata metadata =
-          TracingMetadataUtils.buildMetadata(
-              buildRequestId, commandId, actionKey.getDigest().getHash(), spawn.getResourceOwner());
+          TracingMetadataUtils.buildMetadataWithAttemptNumber(
```

Comment:
Is there a reason we can't change the signature of `buildMetadata`?

Reply:
We can! I thought this would be better to touch less files, but yeah I think this is ugly, I'll just modify `buildMetadata` directly instead.

Reply:
Yeah I think it looks little better this way!